src/store/mod.rs

```diff
@@ -32,7 +32,7 @@ impl EventStoreLockGuard {
 /// An EventStore is responsible for persisting events that an aggregate emits into a database, and loading the events
 /// that represent an aggregate's history from the database.
 #[async_trait]
-pub trait EventStore {
+pub trait EventStore: Sync {
```

Comment:
I *believe* this Sync is also not needed on the trait itself, but only on multi-threaded implementors